### ISSUE TITLE
sentry release failed on GitHub workflow

### ISSUE THREAD

**Issue by MahaZaid** [getsentry/sentry-cli#945]
### Environment

Which SDK and version?
.dotnet core 3

### Steps to Reproduce
I am trying to publish a release through GitHub workflow

```
- uses: actions/checkout@v2
      with:
        fetch-depth: 0

    - name: Create Sentry release
      uses: getsentry/action-release@v1
      env:
        SENTRY_AUTH_TOKEN: ${{ secrets.SENTRY_AUTH_TOKEN }}
        SENTRY_ORG: ${{ secrets.SENTRY_ORG }}
        SENTRY_PROJECT: ${{ env.SENTRY_PROJECT }}
      with:
        environment: ${{ env.EKS_CLUSTER_NAME }}
        version: ${{ env.VERSION }}
```
### Expected Result

publish the release

### Actual Result

<img width="1075" alt="Screen Shot 2021-04-14 at 2 27 37 PM" src="https://user-images.githubusercontent.com/20710308/114710038-91321600-9d2d-11eb-86b9-4e20089a03e9.png">

I am already adding fetch before running the plugin

Is there a way to publish the release without the GitHub plugin ?

**Comment by kamilogorek** [comment]
If `fetch-depth: 0` is set, as you already mentioned, it means that the last release containing associated commits (you can find it in the Releases section of your Sentry account) is using commit hash which has been somehow removed from your git repository. Most likely by either amending or rebasing. And because of that, we cannot find it in the repo and create a valid diff.

The easiest way to fix this is to archive releases that contain non-existing commits.

**Comment by iwakoscott** [comment]
I am not sure if this is similar but, I am seeing `error: release not found` when I run - `sentry-cli releases set-commits --auto $VERSION`. Hitting `https://sentry.io/api/0/organizations/<org_name>/releases/<commit_sha>/` gives me a 404.

**Comment by kamilogorek** [comment]
@iwakoscott it should work not, we had a small hiccup on our end.

**Comment by marandaneto** [comment]
@kamilogorek it should work `now` you meant I believe

**Comment by kamilogorek** [comment]
Details, details. Thanks for the correction :D

**Comment by iwakoscott** [comment]
Thank you! 🙏

**Comment by kamilogorek** [comment]
Released `--ignore-missing` flag in `1.65.0`
